components/live_bracket.py

```diff
@@ -14,7 +14,7 @@ def get_time(file_path: Path) -> datetime.datetime:
 
 def live_bracket():
     home = Path.home()
-    live_path = home / "tourney" / "results_cache" / "Champion_live"
+    live_path = home / "tourney" / "results_cache" / "Legend_live"
```

Comment:
I'll update this at some point when the feature is released, same with live score